### PR TITLE
Broadcast tx over nostr

### PR DESCRIPTION
**Pull request by benthecarman**
Implements https://github.com/nostr-protocol/nips/pull/476

For now we still broadcast to esplora as well because I am pretty sure I am the only one running a server.

Need to make the nostr-sdk more wasm safe

### CODE REVIEW

File: mutiny-core/Cargo.toml [hunk 1]
```diff
@@ -15,7 +15,7 @@ repository = "https://github.com/mutinywallet/mutiny-node"
 [dependencies]
 ln-websocket-proxy = "0.3.0"
 lnurl-rs = { version = "0.2.2", default-features = false, features = ["async", "async-https"] }
-
+nostr-sdk = { version = "0.21.0", default-features = false }
```
Comment (TonyGiorgio) [inline mutiny-core/Cargo.toml:18]: This uses `tokio`, are you sure it's browser safe? Might be wasm safe but idk about `tokio::sync` https://github.com/rust-nostr/nostr/blob/master/crates/nostr-sdk/Cargo.toml#L41

Comment (benthecarman) [reply mutiny-core/Cargo.toml:18]: I tested and was able to broadcast a transaction, will look into where tokio is needed

Comment (benthecarman) [reply mutiny-core/Cargo.toml:18]: looks like it is used for channels

Comment (TonyGiorgio) [reply mutiny-core/Cargo.toml:18]: I didn't think tokio channels were wasm safe. I've just been going with futures.

Comment (TonyGiorgio) [reply mutiny-core/Cargo.toml:18]: Nothing I'm seeing online convinces me that it's truly wasm safe. From documentation, I see that even `tokio::sync::mpsc` will block if there's multiple sends that it needs to do. https://github.com/tokio-rs/tokio/blob/c89406965ffb4a64936d781c556a2c48855dfbdb/tokio/src/sync/mpsc/bounded.rs#L448-L451

Comment (benthecarman) [reply mutiny-core/Cargo.toml:18]: Okay will convert to draft and we can make PRs into rust-nostr

Comment (TonyGiorgio) [reply mutiny-core/Cargo.toml:18]: I could also be wrong, it's just so hard to tell

Comment (benthecarman) [reply mutiny-core/Cargo.toml:18]: We could just implement our own client. All we need is a websocket, won't be very hard.

**Comment by TonyGiorgio** [comment]
Let's go without this

**Pull request closed without merging**